src/components/TodoList/TodoList.tsx

```diff
@@ -1,100 +1,85 @@
 import React from 'react';
+import { Todo } from '../../types/Todo';
+import cn from 'classnames';
 
-export const TodoList: React.FC = () => (
+interface Props {
+  todos: Todo[];
+  selectedTodo: Todo;
+  selectTodo: (event) => void;
```

Comment:
- Specify types for arguments, and fix it in all places
- Prop name for handlers should start from `on` - `onSelectTodo` for example. https://javascript.plainenglish.io/handy-naming-conventions-for-event-handler-functions-props-in-react-fc1cbb791364
- Why does it accept event?